dev/travis/before_install.sh

```diff
@@ -23,4 +23,56 @@ phpenv rehash;
 test -n "$GITHUB_TOKEN" && composer config github-oauth.github.com "$GITHUB_TOKEN" || true
 
 composer config --global http-basic.repo.magento.com "$MAGENTO_USERNAME" "$MAGENTO_PASSWORD"
-git clone https://github.com/magento/magento2
+# TODO: we should specify a particular SHA/branch here
+git clone --depth 1 https://github.com/magento/magento2
```

Comment:
@sivaschenko I think we need to narrow down which version/SHA of `magento2` we clone/use for functional tests. Do you have any opinion on this?

Reply:
I think we should use the latest one